src/features/paste/actions/paste.actions.ts

```
@@ -0,0 +1,1108 @@
+    let hashedPassword: string | undefined;
+    if (validatedInput.password) {
+      hashedPassword = await bcrypt.hash(validatedInput.password, 10);
+    }
+
+    // Calculate expiry date
+    const expiresAt = calculateExpiryDate(validatedInput.expiresIn, isAuthenticated);
+
+    // Generate or validate slug
+    let slug: string;
+    
+    if (validatedInput.customUrl && validatedInput.customUrl.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)